build.sh

```diff
@@ -113,3 +124,27 @@ else
     fi
 fi
 popd > /dev/null # ${FRAMEWORK_DIR}
+
+
+CMAKE_BIN=${UT_CONTROL_LIB_DIR}/host-tools/CMake-3.30.0/build/bin/cmake
```

Comment:
? do you need cmake? For ut-core now? You can't assume you know which directory CMAKE is in.